publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -183,7 +183,7 @@ public void setRabbitConnection(Connection rabbitConnection) {
         this.rabbitConnection = rabbitConnection;
     }
 
-    public void init() throws RemRemPublishException {
+    public void init() {
```

Comment:
Removing RemRemPublishException isn't good idea. How it can be determined that something went wrong? I mean from the code level.

Reply:
If we don't remove  "throws RemRemPublishException"  then application will not start if we  have any issues with Rabbitmq  but  that's  what intended in this ticket to start the application if we any issues with Rabbitmq any way we are catching the Exception so user can go through the logs and check it.

Reply:
This situation is very confusing: init() is called and it successes (no exception thrown). But connection to MB hasn't been established and we have non-working application. Will be the connection re-established when MB is available?
We should analyze call-stack and find a place where to catch the exception and handle it appropriately.

Reply:
Let's suppose MB is down at the start any way application will start and if we publish message at that time then  it will lead to error and after sometime  MB got  up and running and if we publish message after that then  connection to MB established and message sent successfully.